2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_setup.py:_flush():71] setting env: {}
2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220307_100701-218rbyz2/logs/debug.log
2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220307_100701-218rbyz2/logs/debug-internal.log
2022-03-07 10:07:01,898 INFO    MainThread:701905 [wandb_init.py:init():404] calling init triggers
2022-03-07 10:07:01,899 INFO    MainThread:701905 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-07 10:07:01,899 INFO    MainThread:701905 [wandb_init.py:init():460] starting backend
2022-03-07 10:07:01,899 INFO    MainThread:701905 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-07 10:07:01,906 INFO    MainThread:701905 [backend.py:ensure_launched():216] starting backend process...
2022-03-07 10:07:01,909 INFO    MainThread:701905 [backend.py:ensure_launched():222] started backend process with pid: 701914
2022-03-07 10:07:01,910 INFO    MainThread:701905 [wandb_init.py:init():469] backend started and connected
2022-03-07 10:07:01,912 INFO    MainThread:701905 [wandb_init.py:init():533] updated telemetry
2022-03-07 10:07:01,924 INFO    MainThread:701905 [wandb_init.py:init():563] communicating current version
2022-03-07 10:07:02,370 INFO    MainThread:701905 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-07 10:07:02,370 INFO    MainThread:701905 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-07 10:07:02,666 INFO    MainThread:701905 [wandb_init.py:init():606] starting run threads in backend
2022-03-07 10:07:05,300 INFO    MainThread:701905 [wandb_run.py:_console_start():1810] atexit reg
2022-03-07 10:07:05,300 INFO    MainThread:701905 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-07 10:07:05,301 INFO    MainThread:701905 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-07 10:07:05,302 INFO    MainThread:701905 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-07 10:07:05,302 INFO    MainThread:701905 [wandb_init.py:init():633] run started, returning control to user process
2022-03-07 10:07:19,605 INFO    MainThread:701905 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-07 10:07:19,606 INFO    MainThread:701905 [wandb_run.py:_restore():1752] restore
2022-03-07 10:07:21,861 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-07 10:07:22,482 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-07 10:07:22,675 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16040
}

2022-03-07 10:07:22,777 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16040
}

2022-03-07 10:07:22,879 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16040
}

2022-03-07 10:07:22,982 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5257
  total_bytes: 16040
}

2022-03-07 10:07:23,084 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:23,187 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:23,289 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:23,393 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:23,495 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:24,107 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}

2022-03-07 10:07:24,726 INFO    MainThread:701905 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16040
  total_bytes: 16040
}
local_info {
}

2022-03-07 10:07:25,789 INFO    MainThread:701905 [wandb_run.py:_append_history():2130] rendering history
2022-03-07 10:07:25,790 INFO    MainThread:701905 [wandb_run.py:_append_summary():2085] rendering summary
2022-03-07 10:07:25,790 INFO    MainThread:701905 [wandb_run.py:_append_files():2180] logging synced files
